lte/gateway/c/session_manager/test/test_session_state.cpp

```diff
@@ -203,7 +203,7 @@ TEST_F(SessionStateTest, test_marshal_unmarshal) {
   EXPECT_EQ(unmarshaled->get_charging_credit(1, ALLOWED_TOTAL), 1024);
   EXPECT_EQ(unmarshaled->get_monitor("m1", ALLOWED_TOTAL), 1024);
   EXPECT_EQ(unmarshaled->is_static_rule_installed("rule1"), true);
-  EXPECT_EQ(session_state->is_dynamic_rule_installed("rule2"), false);
```

Comment:
I think this probably was a typo at some point. It makes more sense to test the unmarshalled session state, not the original